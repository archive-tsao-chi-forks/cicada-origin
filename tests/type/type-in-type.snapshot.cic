def type: Type = Type

type

def pair_of_types: Type = (Type) * Type

pair_of_types
