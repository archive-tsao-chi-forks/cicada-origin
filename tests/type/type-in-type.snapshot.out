the Type Type
the Type (_: Type) * Type

